feature(404): create default 404 page

cefasgarciapereira opened the pull request:
Este PR implementa a página de erro padrão do site discutido em  #66.

**Preview:**
![image](https://github.com/user-attachments/assets/4ec74fc7-d524-4995-af93-ce8456bb2a71)

Commit:
feature(404): create default 404 page

lfzawacki commented:
Provavelmente seria melhor separar um template para algumas partes da página base (como o sistema de analytics) para que elas possam ser incluídas nesse 404 sem repetição. Fora isso eu não entendi porque no diff modificou todos aqueles ícones.

Eu acho que vou dar uma corrigida nesse negócio dos templates e adaptar teu PR quando tiver um tempinho amanhã! Valeu pela contribuição

omestredosdisfarces commented:
> Fora isso eu não entendi porque no diff modificou todos aqueles ícones.

acho que foi autoformat da IDE. Parece que o tab tá com 4 espaços ao invés de 2 e a ordem dos ícones mudou

cefasgarciapereira commented:
> Provavelmente seria melhor separar um template para algumas partes da página base (como o sistema de analytics) para que elas possam ser incluídas nesse 404 sem repetição. Fora isso eu não entendi porque no diff modificou todos aqueles ícones.
> 
> Eu acho que vou dar uma corrigida nesse negócio dos templates e adaptar teu PR quando tiver um tempinho amanhã! Valeu pela contribuição

E aí, @lfzawacki Prazer! Cara, eu acho que o diff dos ícones tem algo a ver com alguma configuração de lint, talvez algum plugin do vscode. Vou ver se há alguma configuração do meu lado que eu consiga corrigir isso. Fico a disposição aí pra ajudar com o que mais precisar (vi que comentou sobre os templates) Só me pingar. Valeuu 🤘🏻

cefasgarciapereira commented:
> > Fora isso eu não entendi porque no diff modificou todos aqueles ícones.
> 
> acho que foi autoformat da IDE. Parece que o tab tá com 4 espaços ao invés de 2 e a ordem dos ícones mudou

Concordo @omestredosdisfarces . Deve ser algo relacionado a isso. Amanhã eu tento corrigir essa parte e já subo.

Commit:
chore(lint): remove auto-format and fix indentation

Commit:
chore: code format and remove matomo

- Eu havia escrito o código baseado no código da `main`, mas notei depois que no código `novo-site` não há o script do matomo, por isso estou removendo.

Commit:
fix: space between words

lfzawacki commented:
Incorporado junto com https://github.com/soberanatv/soberanatv.github.io/pull/88

Obrigado pela contribuição
